packages/vapor/scss/components/navigation.scss

```diff
@@ -125,10 +125,20 @@
                             justify-content: space-between;
                             height: $navigation-menu-section-height;
                             margin-right: $navigation-right-margin;
+                            padding-right: 2px;
                             padding-left: $navigation-horizontal-space;
                             color: var(--navy-blue-80);
                             font-size: var(--default-font-size);
                             letter-spacing: 0.05em;
+                            border: 2px solid transparent;
+                            border-radius: 8px;
+
+                            &:focus-visible {
```

Comment:
`:focus-visible` doesn't seem supported on Safari, can't we use `:focus`?

Reply:
Using `:focus` in the dev tools in Brave didn't toggle the effect, only `:focus-visible` did. I didn't try on Safari, but I'll try it and add `:focus`

Reply:
Also, Safari doesn't let me inspect the page because it sucks